### PR TITLE
Add Automation ID properties to controls

### LINKED ISSUE #812: Add automation ids to controls
In preparation of automating some of the manual testing, adding automation IDs will make controls easier to identify when doing test runs.

### PR DESCRIPTION
**Pull request by paladique**
Fixes #812

### CODE REVIEW

**Comment by paladique** [comment]
The IDs depend on what is coming from the private repo, which is a submodule. This will cause build errors for someone who doesn't have access. Should we just use a dll, or make the repo public?

**Comment by paladique** [comment]
This commit history is a mess 😞 from fidgeting around to make things work and CI pass. Let me know if you'd prefer I squash anything.

File: src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
              xmlns:sampleData="clr-namespace:GitHub.SampleData;assembly=GitHub.App"
              xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI"
+             xmlns:automation="clr-namespace:GitHub.VisualStudio.TestAutomation;assembly=GitHub.VisualStudio.TestAutomation"
              Background="Beige"
              d:DataContext="{d:DesignInstance Type=sampleData:InfoPanelDesigner, IsDesignTimeCreatable=True}"
              d:DesignHeight="300"
              d:DesignWidth="300"
-             mc:Ignorable="d">
+             mc:Ignorable="d"
+             AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.GitHubToolBar}">
```
Comment (grokys) [inline src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml]: Is this correct? I'd expect `GitHubToolBar` to mean the toolbar in the GitHub pane.

File: src/GitHub.UI.Reactive/Controls/TwoFactorInput.xaml [hunk 2]
```diff
@@ -19,11 +20,11 @@
...
-        <TextBox x:Name="three" />
-        <TextBox x:Name="four" />
-        <TextBox x:Name="five" />
-        <TextBox x:Name="six" Margin="0" />
+        <TextBox x:Name="one" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberOneTextBox}" />
+        <TextBox x:Name="two" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberTwoTextBox}" />
+        <TextBox x:Name="three" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberThreeTextBox}" />
+        <TextBox x:Name="four" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberFourTextBox}" />
+        <TextBox x:Name="five" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberFiveTextBox}" />
+        <TextBox x:Name="six" Margin="0"  AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberSixTextBox}" />
```
Comment (grokys) [inline src/GitHub.UI.Reactive/Controls/TwoFactorInput.xaml]: This strikes me as slightly clumsy. Would it make more sense to add the ID to the container stack panel and then get the textboxes as children of that? Is that even possible? ;)

File: src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml (hunk 1 above)
Comment (paladique) [inline src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml]: Yeah you're right, I think I made a new ID for this but its not reflected in here somehow.

File: src/GitHub.UI.Reactive/Controls/TwoFactorInput.xaml (hunk 2 above)
Comment (paladique) [inline src/GitHub.UI.Reactive/Controls/TwoFactorInput.xaml]: That should be possible! I just grabbed all controls that looked testable and put an ID on it 😬

**Comment by paladique** [comment]
@grokys should be good now! I had to update the assembly resolver to point to the proper dll

**Pull request merged**